source/main.bmx

```diff
@@ -2487,22 +2487,6 @@ Type TSaveGame Extends TGameState
 
 	Global _nilNode:TNode = New TNode._parent
 	Function RepairData(savegameVersion:Int)
```

Comment:
Den Teil mit den reparierten "start/ende" haette man doch drin lassen koennen?

Reply:
Ich hatte übersehen, dass die slots in die Daten übernommen werden (nur entfallenes enabled-Flag). Den Teil stelle ich wieder her.